cmd/postgres_exporter/percona_exporter.go

```diff
@@ -0,0 +1,143 @@
+package main
```

Comment:
Was this file created to make it easier to support upstream changes?

Reply:
yep, to separate our code from upstream